src/renderer/scss/component/_search.scss

```diff
@@ -1,106 +1,105 @@
-.wunderbar {
-  cursor: text;
-  display: flex;
-  flex: 1;
-  min-width: 175px;
-  position: relative;
-  z-index: 1;
+.search {
+  .media {
```

Comment:
The media styles shouldn't change just because they are on the search page. They should be contained to the `_media.scss` file.

Reply:
The media on the search page are not displayed the same as everywhere else. My previous design kept the design uniform but that was unwanted.